src/main/kotlin/org/kamiblue/client/module/AbstractModule.kt

```diff
@@ -53,9 +57,11 @@ abstract class AbstractModule(
 
     fun toggle() {
         enabled.value = !enabled.value
+        if (enabled.value) clicks.value++
     }
 
     fun enable() {
+        clicks.value++
         enabled.value = true
     }
```

Comment:
Why are you increasing clicks twice per enable?

Reply:
What do you mean, the toggle never calls enable and the enable never calls toggle. This was what was causing it not to work earlier. It needs to be incremented if either one is run.

Reply:
Ohh oopsies